### ISSUE TITLE
2 chromosomal fusions from verkko 2.2 assembly not supported by the data

### ISSUE THREAD

**Issue by tbenavi1** [marbl/verkko#294]
Hello,

I have a recent verkko 2.2 assembly with two chromosomal fusions (chr5-chr17, chr8-chr11). However, these translocations are not supported by the long reads or the hic data, from what I can tell. Would you be able to take a look at the assembly? Let me know what files you need. This sample has 40x hifi, 35x ONT-UL, and 25x hi-c data. One of the fusions looks to be a telomere-telomere fusion, and the other looks to be a centromere-telomere fusion. The breakpoints in question are approximately haplotype2-0000066:84375000 and haplotype1-0000025:133700000.

**Comment by skoren** [comment]
I am not sure what would cause this kind of fusion unless at least some of the reads support it. Can you share the noseq.gfa, paths.tsv, colors.tsv, and scfmap files from this asm?

**Comment by tbenavi1** [comment]
Hello, I have added the files here: https://drive.google.com/drive/folders/1-RPypEsl85qqm9B8Gy3ILG_oaD4aq5S5?usp=drive_link

**Comment by skoren** [comment]
The assembly graph is extremely fragmented, much more than I would expect given the coverage. Is this a regular clonal sample? There are over 12k nodes whereas normally we see 2-3k and there is one component connecting most of the chromosomes, also something we don't normally see. I think the joins in question are from Hi-C scaffolding getting confused by the graph structure but the fundamental issue is the very low quality of the graph.

**Comment by tbenavi1** [comment]
Hello, this sample is prepared from PBMCs purified from whole blood and we assume it to be a regular clonal sample. Specifically, it is a normal control for a tumor normal pair (but we have not yet generated sequence data for the tumor). We are investigating if there was anything unusual about this sample during the sequencing preps. Is is possible to see which data type is contributing to the connected component in the graph(e.g. HiFi, ONT, and/or Hi-C)?

**Comment by skoren** [comment]
The base graph is built with HiFi and then resolved with ONT. I'd suspect low coverage or short ONT data. What's your N50 of the ONT data and total coverage vs coverage >100kb?

**Comment by tbenavi1** [comment]
Hello, the total ONT coverage is 35x, the coverage over 100kb is 8x. The N50 is around 60kbp. It looks like the ONT data was from the ligation sequencing kit SQK-LSK114 and not the ultra long kit.

**Comment by tbenavi1** [comment]
Yes, I'm very curious. I understand that scaffolding with a fragmented graph is a difficult problem, but I would hope that it wouldn't put two pieces together that shouldn't be put together. Thanks for taking a look.

**Comment by Dmitry-Antipov** [comment]
Hi,
Both of that fusions are caused not by Hi-C scaffolding but by assembly graph structure - excessive joins near coverage breaks leaves only one haplotype-preserving path through the assembly graph, which is actually an erroneous fusion.

Seems that for your graph for some reasons telomeres are often connected to the middle of other chromosomes, and we do not see this things normally.  Can you also share 8-hicPipeline/unitigs.telo ?

**Comment by tbenavi1** [comment]
Hello, I have added unitigs.telo to the folder.
